public/docs/features.md

```diff
@@ -325,7 +325,7 @@ gantt
   section Section
   A task: a1, 2014-01-01, 30d
   Another task: after a1, 20d
-  
```

Comment:
Did you mean to change this?

Reply:
No, this was done automatically by VSCode while editing the page.

Reply:
Then I think you should remove this…

Reply:
Fixed.